mysql-test/suite/rpl/t/rpl_bug85141.test

```diff
@@ -0,0 +1,127 @@
+#
+# Bug #85141 "Write/fsync amplification w/ duplicate GTIDs"
+#
+
+--source include/not_group_replication_plugin.inc
+--source include/have_innodb.inc
+--source include/have_gtid.inc
+
+--echo #
+--echo # Setting up MSR server topology
+--echo #
+--echo # server_1(A) -------------- server_3(C)
+--echo #            \              /
+--echo #             - server_2(B)-
+--echo #
+--let $rpl_multi_source = 1
+--let $rpl_topology = 1->2,2->3,1->3
+--source include/rpl_init.inc
+
+--echo #
+--echo # Step 1: stopping slave on server_2(B)
+--echo #
+--let $rpl_connection_name = server_2
+--source include/rpl_connection.inc
+STOP SLAVE;
```

Comment:
Async always looks suspicious in MTR - use include/stop_slave.inc include/start_slave.inc here and everywhere?